<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSimilarityView</class>
 <widget class="QMainWindow" name="ImageSimilarityView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitterBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="QWidget" name="ImageWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitterImage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>3</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ac_Save"/>
    <addaction name="separator"/>
    <addaction name="ac_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Algorithm">
    <property name="title">
     <string>Algorithm</string>
    </property>
    <widget class="QMenu" name="menu_Histogram">
     <property name="title">
      <string>Histogram</string>
     </property>
     <addaction name="ac_Histogram"/>
     <addaction name="ac_HistogramSub"/>
    </widget>
    <widget class="QMenu" name="menu_SIFT">
     <property name="title">
      <string>SIFT</string>
     </property>
     <addaction name="ac_Test"/>
    </widget>
    <widget class="QMenu" name="menu_HASH">
     <property name="title">
      <string>HASH</string>
     </property>
     <addaction name="ac_Test2"/>
    </widget>
    <addaction name="menu_Histogram"/>
    <addaction name="menu_SIFT"/>
    <addaction name="menu_HASH"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ac_About_Contents"/>
    <addaction name="ac_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Algorithm"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ac_Save"/>
   <addaction name="separator"/>
   <addaction name="ac_Histogram"/>
   <addaction name="ac_HistogramSub"/>
   <addaction name="separator"/>
   <addaction name="ac_About"/>
   <addaction name="ac_Exit"/>
  </widget>
  <action name="ac_Save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="ac_Histogram">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="ac_HistogramSub">
   <property name="text">
    <string>Aptive Histogram Equalization</string>
   </property>
  </action>
  <action name="ac_Test">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="ac_Test2">
   <property name="text">
    <string>test2</string>
   </property>
  </action>
  <action name="ac_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="ac_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="ac_About_Contents">
   <property name="text">
    <string>About Contents</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
